src/main.py

```diff
@@ -170,7 +170,12 @@ def utimens(self, path, times=None):
 
```

Comment:
please add a comment or something that using this will throw massive amounts of error since this is using your own path ( or remove it)